rbac/server/api/proposals.py

```diff
@@ -192,5 +191,10 @@
     else:
         user_resource = await fetch_user_resource(conn, proposal_resource.get("object"))
         proposal_resource["approvers"] = [user_resource.get("manager")]
+    for i in range(len(proposal_resource["approvers"])):
```

Comment:
It's usually against practice to modify the iterating variable during a loop. you might want to use a while loop here instead.

Reply:
I updated it please check it